codespell_lib/data/dictionary.txt

```diff
@@ -10074,6 +10074,8 @@ dnamically->dynamically
 dne->done
 do'nt->don't
 doalog->dialog
+doamin->domain
+doamins->domains
```

Comment:
Perhaps add the subdomain(s) variants of these too (if they don't already exist).

Reply:
This is indeed a common mispelling (https://grep.app/search?q=subdoamin), which was not present. Added, thanks!